rts/Rendering/Env/Particles/ProjectileDrawer.cpp

```diff
@@ -693,6 +737,9 @@ void CProjectileDrawer::DrawProjectilesMiniMap()
 		}
 	}
 
+	// TODO "visibleParticles" has only camera visible particles
+	// if we remove modellessProjectiles we can get all projectiles from:
+	// "projectileHandler->GetActiveProjectiles()"
```

Comment:
It isn't clear what the task in the TODO is. It seems to be more thoughts on a subject rather than a task.